### DISCUSSION TITLE
Change image scale

### DISCUSSION THREAD

**Discussion by thiagobsilva28** [fo-dicom/fo-dicom#1378] [Q&A]
Good afternoon!

How is it possible to reduce the pixel density, pixel width and pixel height of a DICOM image using fo-dicom?

I'm needing to scale down the image to display the image on devices that cannot render images at a very high resolution.

Has anyone ever needed to do this image resizing?

Thanks!

**Comment by gofal** [comment]
actually fo-dicom has nothing like that included. I often needed to resize the images, but then I only stored the resized ready rendered images. never made a dicom-file from that reduced image again. But keep in mind: if you reduce the size, then you throw away some information and you therefore should create an entirely new file, with new SopInstanceUIDs, updated ImageComment, updated timestamps of aquisition, you should add a ReferencedImageSquence to refer to the original image, etc.

if you do not have the need to generate some dicom file, but you want do to this scaling for your own applications, then i would recommend, to render the image and then continue to work with png/jpg etc of that rendered image in some reduced sizes.
